SamMou.Api/appsettings.Development.json

```diff
@@ -10,5 +10,6 @@
     "Endpoint": "https://sammoucosmos.documents.azure.com:443/",
     "PrimaryKey": "",
     "DatabaseName": "SamMou"
-  }
+  },
+  "APPINSIGHTS_INSTRUMENTATIONKEY": ""
```

Comment:
different format than other constants?

Reply:
cause this one only single string not a structure

Reply:
okay